rainbow-me/rainbow
# Swaps: Fix select currency scroll on Android

Fixes TEAM2-270

## What changed (plus any additional context for devs)

Fixing select currency scroll on Android

## Screen recordings / screenshots


https://user-images.githubusercontent.com/7336481/179878231-5ca3670d-dd7d-4657-9598-e79a06a09275.mp4



## What to test

- Make sure scroll works on Android
- Make sure scroll works on iOS

## Final checklist
